ArjunaJTA/jta/tests/classes/com/hp/mwtests/ts/jta/recovery/RecoveryScanTest.java

```diff
@@ -28,8 +28,8 @@
 import com.arjuna.ats.internal.jta.recovery.arjunacore.XARecoveryModule;
 import com.arjuna.ats.internal.jta.transaction.arjunacore.TransactionImple;
 import com.arjuna.ats.jta.common.jtaPropertyManager;
-import com.arjuna.ats.jta.logging.RecoveryRequired;
 import com.arjuna.ats.jta.logging.jtaLogger;
+import com.arjuna.ats.jta.recovery.RecoveryRequired;
```

Comment:
Where is this used?

Reply:
line no. 95

Reply:
[Line 95](https://github.com/jbosstm/narayana/pull/1600/files#diff-24734fbaf2c75b67cc6c5de0d7795472R95) is a blank line.

Reply:
I see that your latest push has fixed this so marking as resolved.